vector/src/main/java/im/vector/app/features/media/ImageContentRenderer.kt

```diff
@@ -121,7 +125,8 @@ class ImageContentRenderer @Inject constructor(private val activeSessionHolder:
                     .load(data)
         } else {
             // Clear image
-            val resolvedUrl = activeSessionHolder.getActiveSession().contentUrlResolver().resolveFullSize(data.url)
+            val resolvedUrl = resolveUrl(data)
+                    ?: data.url.takeIf { data.allowNonMxcUrls }
```

Comment:
This fallback is already done in `fun resolveUrl()`, no?

Reply:
no not anymore , can't be done by default

Reply:
Are you sure?

Reply:
oh yes got it